app/views/works/main.html.erb

```diff
@@ -0,0 +1,34 @@
+<main>
+  <section class="spotlight">
+    <h2 class="spotlight__header">
+      <span class="spotlight__header--prefix">
+        Media Spotlight
+      </span>
+      <% spotlight = @works.sample%>
+      <a class="spotlight_link-to" href="works/<%=spotlight.id%>"><%=spotlight.title%></a>
+      by <%=spotlight.creator%></h2>
+      <p class="spotlight__description"><%=spotlight.votes.count%> votes - <%=spotlight.description%></p>
+    </section>
+    <hr class="root__hr">
+    <section class="top-ten__container">
+      <%['movie', 'book', 'album'].each do |category|%>
+      <section class="top-ten__list-container">
+        <h2 class="top-ten__header">
+          Top <%=category.capitalize%>s</h2>
+          <ul class="list-group top-ten__list">
+            <%@works = Work.sort_by_votes(category)%>
```

Comment:
This works well, but this kind of selection and directly accessing the database should go in the controller.  That way the view can simply take in data passed by the controller. It means the controller could be switched in how it functions without the view being updated.  

This also applies to to spotlight.